javaparser-core/src/main/java/com/github/javaparser/resolution/types/ResolvedReferenceType.java

```diff
@@ -170,6 +170,11 @@ public String describe() {
      */
     public abstract ResolvedType transformTypeParameters(ResolvedTypeTransformer transformer);
 
+    /**
+     * @return A copy of the current reference type, without type parameters.
+     */
+    public abstract ResolvedType asRawType();
```

Comment:
toRawType() method does not do what you want?

Reply:
I'm blind, let me fix that.